src/examples/chord/chord.ts

```diff
@@ -128,8 +128,6 @@ const vis: ChordVisualization = {
     const innerRadius = outerRadius - thickness
 
     // Stop if radius is < 0
-    // TODO: show warning to user ???
-    // TODO: Set a min-radius ???
     if (innerRadius < 0) return
```

Comment:
What happens if inner radius is less than 0?

Reply:
Here is the repro on learn beta: https://learnbeta.looker.com/explore/customer_1_model/order_items?qid=vWIhlV7cV55HJtjZfy9J5y&toggle=fil,vis

Reply:
@RichardCzechowski I believe this just hides the vis if the viewport is too small. I don't think the viewport will actually get that small since the Explore UI has some minimum widths and heights, but I'm afraid to take this out in case it's for something I didn't test.